### PR TITLE
Fixes added

### PR DESCRIPTION
**Pull request by VeryOldLady**

### CODE REVIEW

File: src/services/auth/auth_test.go [hunk 1]
```diff
@@ -28,18 +30,33 @@ func TestLogin(t *testing.T) {
...
+
 	data := url.Values{}
-	data.Add("login", "admin")
-	data.Add("password", "admin")
+	data.Add("login", expetedLogin)
+	data.Add("password", expetedPass)
 	originalDB := db
 	defer func() { db = originalDB }()
+	var err error
 	db, mock, err = sqlmock.New()
```
Comment (mshto) [inline src/services/auth/auth_test.go]: Do you use db variables in your tests?

**Comment by mshto** [comment]
In auth.go
var	GetUserByLogin  = models.GetUserByLogin  
Could you just call method models.GetUserByLogin() ?

**Comment by VeryOldLady** [comment]
how will I mock it in tests?

**Comment by mshto** [comment]
Yaromyr can help you with it.

File: src/services/auth/auth_test.go (hunk 1 above)
Comment (Nastya-Kruglikova) [reply src/services/auth/auth_test.go]: @mshto is correct. Handler auth should know nothing about DB. If you need to mock function from models, mock it in models

**Pull request merged**